fedocal/fedocallib/__init__.py

```diff
@@ -666,7 +666,7 @@ def retrieve_meeting_to_remind(session, offset=30):
     return meetings
 
 
-def add_meeting_to_vcal(ical, meeting):
+def add_meeting_to_vcal(ical, meeting, enable_reminder=False, reminders=[]):
```

Comment:
Thinking out loud: do we need both arguments?
We could use only one: the reminder to set (either it is None, or there is one)